web/client/components/style/vector/Manager.jsx

```diff
@@ -64,25 +69,15 @@ class Manager extends React.Component {
         style: {},
         defaultShape: DEFAULT_SHAPE,
         symbolsPath: DEFAULT_PATH,
-        defaultSymbol: {
-            iconAnchor: [0.5, 0.5],
-            anchorXUnits: 'fraction',
-            anchorYUnits: 'fraction',
-            color: "#000000",
-            fillColor: "#000000",
-            opacity: 1,
-            size: 64,
-            fillOpacity: 1
-        },
+        defaultShapeSize: 64,
+        defaultShapeFillColor: '#000000',
+        defaultShapeStrokeColor: '#000000',
+        defaultPointType: 'marker',
```

Comment:
to me the defaultPointType prop seems not used, if it is the case should be removed